conda/environments/all_cuda-120_arch-x86_64.yaml

```diff
@@ -0,0 +1,45 @@
+# This file is generated by `rapids-dependency-file-generator`.
+# To make changes, edit ../../dependencies.yaml and run `rapids-dependency-file-generator`.
+channels:
+- rapidsai
+- rapidsai-nightly
+- conda-forge
+- nvidia
+dependencies:
+- bokeh>=2.4.2,<=2.5
+- cuda-version=12.0
+- cudf==23.8.*
+- cugraph==23.8.*
```

Comment:
cugraph isn't ready for CUDA 12 yet. Should we hold off on merging this PR until the dependencies are ready? Or do we ignore that because it's only needed for notebooks? The environment file won't be solvable until that is ready, so we might need a workaround in `dependencies.yaml` (e.g. don't generate the CUDA 12 environment, or drop cugraph from CUDA 12 environments, etc.).

Reply:
If it is only needed for notebooks, would suggest we go ahead and get this PR in. We can test notebooks later

Reply:
ok, sounds good